src/content/docs/style-guide/images/create-videos.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Create videos
+tags:
+  - Tech writer style guide
+  - Processes and procedures
+redirects:
+---
+
+There are many ways to create videos, but here are some tips that will likely apply to a lot of video production projects. 
+
+## Before you start [#before-you-start]
+
+Before you start, talk to the Nerd Bytes team and make sure they're not working on a similar video. 
+
+## Scripting tips [#script-tips]
+
+Try to keep videos under two minutes. If your script is longer than that, consider breaking it up into more than one video. 
+
+You can estimate your video length by doing a leisurely read-through of your script. 
+
+## Tips for recording [#recording-tips]
+
+Tips for using Camtasia to record your screen capture and voice-over audio: 
+
+* Record in a quiet room, using some sort of external microphone (for example, Apple headphones). Don't use the built-in computer microphone. 
+* Record several seconds of room tone (the sound of the empty room). This can be used to fill in gaps or cover up noises in the audio. This is better than using silence because going to complete silence can be jarring. 
+* You can easily take out pauses during editing: this means you can pause to prepare for the next sentence or section as long as you want.
+* Instead of reading the whole thing through multiple times, consider trying to read it once but, for sections you think weren't done well, immediately repeat that section again. This can be easier to edit than doing entire-script takes. 
```

Comment:
How about this revision:

OLD: Instead of reading the whole thing through multiple times, consider trying to read it once but, for sections you think weren't done well, immediately repeat that section again. This can be easier to edit than doing entire-script takes.

NEW: Instead of reading the whole thing through multiple times, consider trying to read it once. For sections you think weren't done well, immediately repeat that section again. This can be easier to edit than doing entire-script takes.